Deploy/db/registry.sql

```diff
@@ -65,31 +72,22 @@ create table project (
 );
 
 insert into project values 
-(null, 1, 'library', NOW(), 0, 1);
+(null, 1, 'library', NOW(), NOW(), 0, 1);
```

Comment:
The field 'Project ID' is a not null field, it should not insert a null value as default.